ADQL.tex

```diff
@@ -1409,6 +1417,34 @@ \subsubsection{Overview}
     \item REGION
 \end{itemize}
 
+\subsubsection{Language feature}
+\label{sec:functions.geom.feature}
+
+All functions described in this section use the following IVOID:
+
+\begin{verbatim}
+    ivo://ivoa.net/std/tapregext#features-adqlgeo
+\end{verbatim}
+    
+Each geometrical function is declared using this IVOID and its name.
+See \SectionRef{sec:capabilities} for more details.
+
+All other optional features have an IVOID starting with:
+
+\begin{verbatim}
+    ivo://ivoa.net/std/tapregext#features-adql-
+\end{verbatim}
+
+Note the ending hyphen. The IVOID for the geometry features does not have this
+hyphen. This is not a typo. It actually comes from the \TAPRegSpec{} which
+originally defined this IVOID. At that time, it was not clear yet that other
+language features would be created for ADQL. Thus, it lasted until the first
+ADQL-2.1 implementations appeared in the IVOA registry and before ADQL-2.1
+became an IVOA Recommendation. Because a lot of implementations were already in
+a production state when this inconsistency with the other ADQL features was
+detected, it had been decided to not break these implementations and to keep
+this IVOID as such, only for geometry features.
```

Comment:
I would be inclined to cut back on the historical detail here, since it's not really necessary for understanding the standard.  I'd suggest just writing something like "For historical reasons the IVOID for the geometry features does not have this hyphen" and leaving it at that.  But I leave it to @gmantele to decide whether to follow that suggestion.

Reply:
Done. I still want to keep a small explanation of why this difference. So I left the TAPRegExt reference.
Is it better?